R/checkStartEndDate.r

```diff
@@ -8,10 +8,6 @@
 #' @keywords WRTDS flow
 #' @return vector where first value is startDate, second is endDate
 #' @export
-#' @examples
-#' startDate <- '1985-01-01'
-#' endDate <- '1990-01-01'
-#' checkStartEndDate(startDate, endDate)
```

Comment:
I knew I deleted some example by mistake. My problem was I did a `git push upstream ffn` instead of origin, and got all confused